astropy/io/ascii/ui.py

```diff
@@ -223,18 +223,18 @@ def read(table, guess=None, **kwargs):
     names : list
         List of names corresponding to each data column
     include_names : list
-        List of names to include in output (default= ``None`` selects all names)
+        List of names to include in output (default=None selects all names)
     exclude_names : list
         List of names to exclude from output (applied after ``include_names``)
     fill_values : dict
         specification of fill values for bad or missing table values
     fill_include_names : list
-        List of names to include in fill_values (default= ``None`` selects all names)
+        List of names to include in fill_values (default=None selects all names)
     fill_exclude_names : list
         List of names to exclude from fill_values (applied after ``fill_include_names``)
     fast_reader : bool or dict
         Whether to use the C engine, can also be a dict with options which default to ``False``
-        (default= ``True``); parameters for options dict:
+        (default=True); parameters for options dict:
```

Comment:
Here, the default is actually set in the code; I would use "(default: `True`)", to enable the sphinx interlink.